server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -53,7 +53,7 @@ function getRoomId(room: IRoom) {
 
 const getSocketConnectThrottleId = (tenantId: string) => `${tenantId}_OpenSocketConn`;
 
-const getSubmitOpThrottleId = (clientId: string) => `${clientId}_SubmitOp`;
+const getSubmitOpThrottleId = (tenantId: string, clientId: string) => `${clientId}_${tenantId}_SubmitOp`;
```

Comment:
Nit: Since the naming is clientId_tenantId, it is better that the parameters follow the same order